docker/linux-image.sh

```diff
@@ -171,12 +171,14 @@ main() {
         sharutils \
         gnupg
 
-    # amd64 has conflicting versions of the packages installed, so
+    # image arch has conflicting versions of the packages installed, so
     # we need to remove the system installs later. since apt relies
     # on these packages, we need to download them and reinstall
     # using dpkg later, since we cannot redownload via apt.
+    local image_arch
```

Comment:
```suggestion
    local host_arch
```

Reply:
Should this really be `host_arch` and not `image_arch`? We could also just use `TARGETARCH` here likely, but I feel `image_arch` is slightly more specific since it clarifies this is the architecture of the generated Docker image, and not the container engine or anything else.

Reply:
sounds good, if it's kept though I think `dpkg_arch` makes sense, dunno why I thought `host_arch` was a good alternative